index.css

```diff
@@ -1,13 +1,129 @@
-.hero {
-  background-image: url("./assets/01_Main_Hero_Mobile.jpg");
-  background-size: cover;
-  background-position: center 33%;
-  height: 100vh;
+
+
+/*////////////////// NAV-BAR*/
+.nav-bar{
+  padding: 0 10px ;
   width: 100%;
+  height: 100px;
+  position: fixed;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  }
+  li,h2{
+    color: blanchedalmond;
+  }
+  .nav-container{
+    flex-basis: 50%;
+    display: none;
+    justify-content: space-between;
+
+  }
+  .nav-menu{
+    flex-basis: 40%;
```

Comment:
flex-basis is (what I have seen so far) rather unusual. Most use the shorthand-property `flex: n` to define the sizes.
Of course your solution is correct, but only for your information:
As it is quite nicely illustrated, how flex works you can see for example here: https://facebook.github.io/react-native/docs/flexbox.